src/periphery/FeeSplitter.sol

```diff
@@ -1,11 +1,15 @@
 pragma solidity ^0.8.18;
 
 import {Ownable2Step} from "openzeppelin/access/Ownable2Step.sol";
-import "v2-core/src/interfaces/ISubAccounts.sol";
+import {IManager} from "v2-core/src/interfaces/IManager.sol";
+import {ISubAccounts} from "v2-core/src/interfaces/ISubAccounts.sol";
 
 contract FeeSplitter is Ownable2Step {
   ISubAccounts public immutable subAccounts;
   uint public immutable subAcc;
+  uint private splitPercent;
+  uint private accountA;
+  uint private accountB;
```

Comment:
would make these all public so they can be queried externally (making them public provides a view function)